include/platform/clinician_input.h

```diff
@@ -13,3 +13,13 @@
  *
  */
 status_t get_respiratory_rate(uint32_t * RR);
+
+/*
+ * Updates <selected inspiratory time> variable insp_time 
+ * when a valid value was read from the clinician inputs
+ * 
+ * @return STATUS_OK if <selected inspiratory time> is valid 
```

Comment:
Including "valid" and "invalid" here is confusing. If this is just a getter, it should make no claims about validity of the value. This should be checked upon input from the clinician, i.e., it should be impossible to enter an invalid value. Do we have SRSs to handle this?

Reply:
see answer above https://github.com/COSMIC-medical/gVent/pull/80#discussion_r528891897